src/main/java/com/checkmarx/flow/service/SastScanner.java

```diff
@@ -106,19 +131,22 @@ public ScanResults scan(ScanRequest scanRequest) {
     }
 
     private ScanResults getScanResults(ScanRequest scanRequest, Integer projectId, Integer scanId) throws CheckmarxException {
-        ScanResults scanResults = null;
-        BugTracker.Type bugTrackerType = bugTrackerEventTrigger.triggerBugTrackerEvent(scanRequest);
-        if (bugTrackerType.equals(BugTracker.Type.NONE)) {
-            scanDetails = handleNoneBugTrackerCase(scanRequest, null, scanId, projectId);
+        if(!cxProperties.getEnablePostActionMonitor()) {
```

Comment:
I think you can combine the bugTrackerNone case with the postActionMonitor in the same condition, since both cases not waiting for results and returning empty scan results

Reply:
We still need to capture information about the scan thats not available on the return. Originally I was able to suck most of the info from the report but pull requests change the dynamic a little. That's the information going into the comment. The bug tracker None item is an option but I still need to know which bug tracker the setup is using.